cpp/src/io/parquet/experimental/hybrid_scan.cpp

```diff
@@ -74,6 +74,18 @@ size_type hybrid_scan_reader::total_rows_in_row_groups(
   return _impl->total_rows_in_row_groups(input_row_group_indices);
 }
 
+std::vector<cudf::size_type> hybrid_scan_reader::filter_row_groups_with_byte_range(
```

Comment:
```suggestion
std::vector<size_type> hybrid_scan_reader::filter_row_groups_with_byte_range(
```
Consistency with `cudf::host_span` parameter type.